components/Home/sidebar/RhHomeSidebarBountiesSection.tsx

```diff
@@ -0,0 +1,92 @@
+import { css } from "aphrodite";
+import { emptyFncWithMsg } from "~/config/utils/nullchecks";
+import { fetchOpenBounties } from "./api/fetchOpenBounties";
+import { ReactElement, useEffect, useState } from "react";
+import { SideColumnTitle } from "~/components/Typography";
+import { styles } from "./styles/RhHomeRightSidebarStyles";
+import HubEntryPlaceholder from "~/components/Placeholders/HubEntryPlaceholder";
+import ReactPlaceholder from "react-placeholder/lib";
+import BountiesSidebarItem from "./SidebarItems/BountiesSidebarItem";
+
+type PaginationInfo = { isFetching: boolean; page?: number };
+
+const useEffectFetchOpenBounties = ({
+  paginationInfo,
+  setOpenBounties,
+  setPaginationInfo,
+}: {
+  paginationInfo: PaginationInfo;
+  setOpenBounties: (bounties: any) => void;
+  setPaginationInfo: (paginationInfo: PaginationInfo) => void;
+}): void => {
+  const { isFetching, page } = paginationInfo;
+  useEffect((): void => {
+    if (isFetching) {
+      fetchOpenBounties({
+        onSuccess: (bounties: any) => {
+          // TODO: calvinhlee deal with page when supported by BE
+          setPaginationInfo({ isFetching: false, page });
+          setOpenBounties(bounties);
+        },
+        onError: emptyFncWithMsg,
+      });
+    }
+  }, [isFetching]);
+};
+
+export default function RhHomeSidebarBountiesSection(): ReactElement {
```

Comment:
Can we name `BountiesSidebar` or `HomeBountiesSidebar` ? I don't think the `Rh` is needed.

Reply:
this component is not a sidebar. It's a section / item to be used on a sidebar. I'll remove `Rh` prefix